extensions/sinks/influx/influx.go

```diff
@@ -76,26 +76,28 @@ func (m *influxSink) Configure(props map[string]interface{}) error {
 	}
 	if i, ok := props["databasename"]; ok {
 		if i, ok := i.(string); ok {
-			m.databasename = i
+			m.databaseName = i
 		}
 	}
 	if i, ok := props["tagkey"]; ok {
 		if i, ok := i.(string); ok {
-			m.tagkey = i
+			m.tagKey = i
 		}
 	}
 	if i, ok := props["tagvalue"]; ok {
 		if i, ok := i.(string); ok {
-			m.tagvalue = i
+			m.tagValue = i
 		}
 	}
 	if i, ok := props["fields"]; ok {
 		if i, ok := i.(string); ok {
 			m.fields = i
 		}
 	}
-	if _, ok := props["dataTemplate"]; ok {
-		m.hasTransform = true
+	if i, ok := props["dataTemplate"]; ok {
+		if i.(string) != "" {
```

Comment:
i.(string) may panic if it is not a string, shall we add the check also?

Reply:
@ngjaying You are right. We need to judge whether this is a string first.I will perfect it.